assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```
@@ -376,7 +376,11 @@ storage:
                   nameservers=$out
                   break
               fi
-              echo "Failed with exit code $ret: $out"
+              if [[ $out = "" ]]; then
```

To be consistent with the line above:
```suggestion
              if [[ "$out" = "" ]]; then
```

---

Indeed, sorry about that. Fixed.